universal-repository-labels.json

```diff
@@ -113,5 +113,10 @@
     "name": "type: support",
     "color": "#ffff00",
     "description": "Request for help using the project."
+  },
+  {
+    "name": "type: priority support",
+    "color": "#ff0000",
+    "description": "Get help within a workday. Only available for Gold/Platin/Diamond tier or dedicated support contract."
```

Comment:
Two issues:
- Is "Platin" correct? From the context, I would expect "Platinum", but there's certainly no reason it couldn't be named "Platin".
- This description exceeds the 100 character maximum limit, so the workflow will fail to add it to the repositories.
 
A soft limit you might also consider when writing label descriptions, though likely not relevant in this particular case: from my experiments, only the first 44 characters of the description are shown in the label menu the maintainer picks labels from. In cases where the meaning of a label is not clear from the name and first 44 characters of the description, this could cause confusion.

Reply:
Excellent catch and a classic German/English translation mistake :+1: I was also not aware of the soft (44) and hard (100) character limits, will correct it asap. Currently I'm quite glad I asked for your review :sweat:

Reply:
About the soft limit. I did some more experimentation and found that it appears to be based on the actual width rather than the number of characters, so if the description uses wider characters then you can fit less, and vice versa. But I think ~45 is a reasonable average to keep in mind while writing descriptions.

Reply:
Question: Now that I fixed the entry, shouldn't I be able to use that label for this PR?

Reply:
No, because it pulls the label data file from the `main` branch:
https://github.com/107-systems/.github/blob/164362bd0d3e5a65460d9bc4b95fd1d956642fed/.github/workflows/manage-labels.yml#L31
It's a bit silly that it downloads a file that's already in this repository. That is because it's using the same general purpose workflow that is used in all the repositories.

Reply:
Makes sense ;) Thank you, will proceed merging now.